### ISSUE TITLE
Update CocoaPods version

### ISSUE THREAD

**Issue by Elipasousa** [TomThorpe/TTRangeSlider#47]
CocoaPods version does not have labelPadding property yet

**Comment by waynemock** [comment]
I'd also like the Accessibility work that's in pull request #41 . My App that uses TTRangeSlider via CocoaPods needs to be ADA AA compliant and I need VoiceOver to interact with it. Currently I does not see it as an accessible control.

**Comment by TomThorpe** [comment]
Hey this should be done and the accessibility work is also in there. Apologies for taking so long!
